spring-cloud-dataflow-apps-plugin/spring-cloud-dataflow-apps-generator-plugin/src/main/resources/template/app-pom.xml

```diff
@@ -141,46 +141,21 @@
 				{{#app.bootPluginConfiguration}}
 				<configuration>
 					{{this}}
+					{{#app.containerImage.enableMetadata}}
+					<image>
+						<name>{{app.containerImage.orgName}}/${project.artifactId}:{{app.containerImage.tag}}</name>
+					</image>
+					{{/app.containerImage.enableMetadata}}
```

Comment:
Do we need to specify any of the options [like before](https://github.com/spring-cloud/stream-applications/pull/548/files#diff-bdd0d0a43457d0ceee46961f59123580d51d87a88e0245d09d0e99d623fe5c1cL160-L161)? Or does the boot image already set these?

Reply:
Added tests

Reply:
Boot Build Image requires all labels in one environment variable. The JSON breaks the parsing of this.